context/brand-context/springer/scss/30-mixins/typography.scss

```diff
@@ -60,3 +60,11 @@
 	font-weight: $context--font-weight-bold;
 }
 
+%u-h5 {
```

Comment:
an `@extend` … "Now, that's a Sass function I've not heard in a long time. A long time."